### DISCUSSION TITLE
I have an empty Matrix, is there a process to shift that into a Super Table field.

### DISCUSSION THREAD

**Discussion by portedison** [verbb/super-table#544] [General]
I've been looking at moving the data in the YAML files, if that the right approach.

(New to Craft)

**Comment by engram-design** [comment]
Super Table fields cannot be converted to Matrix fields (or vice-versa), sorry.

**Reply by portedison** [reply]
Thanks, I wasn't very clear. I have a matrix that is a top level parent, but I would like it to be a child of another SuperTable. 
e.g. 

From this :
My Matrix Field
- Child Field
- Child Field

To this :
My Super Table
- My Matrix Field
  - Child Field
  - Child Field

Do I just shuffle around the YAML files/references. I don't have any data in the tables yet.
